CMakeModules/InternalUtils.cmake

```
@@ -177,7 +177,7 @@ macro(arrayfire_set_cmake_default_variables)
     set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${ArrayFire_BINARY_DIR}/bin)
   endif()
 
-  if(APPLE)
+  if(APPLE AND (NOT DEFINED CMAKE_INSTALL_RPATH))
     set(CMAKE_INSTALL_RPATH "/opt/arrayfire/lib")
```

@umar456, does this work?

```suggestion
  if(APPLE)
    list(APPEND CMAKE_INSTALL_RPATH "/opt/arrayfire/lib")
```